infra/terraform/test_fixtures/terraform-google-forseti.tf

```diff
@@ -8,6 +8,7 @@ locals {
   forseti_host_project_required_roles = [
     "roles/compute.networkAdmin",
     "roles/compute.securityAdmin",
+    "roles/compute.instanceAdmin.v1",
```

Comment:
is there a reason to not just provide Compute Admin?

Reply:
Nope, we can simplify this.

Reply:
Resolved by https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/97/commits/f93a9321672dca680f71e7d140e020cadb0537fb.